src/main/scala/com.snowplowanalytics/schemaguru/generators/JsonSchemaGenerator.scala

```diff
@@ -242,7 +291,12 @@ object JsonSchemaGenerator {
      * @return JsonSchemaType with recognized properties
      */
     def enrichString(value: String) = {
-      guessFormat(value, formatSuggestions) match {
```

Comment:
@chuwy  Sure thing. Strings that are found to be of  `short`, `int`, `long`  are all json `number` json schema type. Thus, the format value can be used for additional schema validations of the `number` json type.

Yeah, I was not going of a json spec. From looking at the code, it seemed the best way to capture specific ranges of numbers.